dtls/client-dtls-shared.c

```diff
@@ -67,13 +65,6 @@ int dtls_sendto_cb(WOLFSSL* ssl, char* buf, int sz, void* ctx)
                   (const struct sockaddr*)&shared->servAddr, shared->servSz);
 }
 
-
-static int min(int a, int b)
```

Comment:
Let's go ahead and leave min() in here.  I think this one makes it easier for people to read, as min() is a standard function anyways.